types/node/index.d.ts

```diff
@@ -1514,7 +1543,6 @@ declare module "https" {
         ciphers?: string;
         honorCipherOrder?: boolean;
         requestCert?: boolean;
-        rejectUnauthorized?: boolean;
```

Comment:
omg. totally mistake. should remove from `ClientRequestArgs`. Must be tired. lol